specification/cognitiveservices/data-plane/LUIS/Programmatic/v2.0/LUIS-Programmatic.json

```diff
@@ -6069,6 +6302,20 @@
         }
       }
     },
+    "OperationStatus": {
+      "description": "Response of an Operation status.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
```

Comment:
Can you make an enum with the expected values? Are these all 'success' conditions?

Reply:
We've seen some operation return "FAILED" and "Failed". I'm not sure if using an enum wont cause an issue when reading those values. Can we leave this as string?

Reply:
you can use  `x-ms-enum` with `modelAsString` : `true` and unknown values are tolerated, and can be expanded without breaking changes later.

Knowing what's available will help people consume the service easier.

``` json
"enum" : [ "Failed", "FAILED", "Success" ], 
"x-ms-enum" : { 
  "modelAsString": "true",
  "name": "OperationStatusType"
}
```

Reply:
Perfect! It generates proper documentation with possible values, and Constants for referencing the values